variants/frontend-base/template.rb

```diff
@@ -9,14 +11,28 @@
 apply "config/template.rb"
 apply "app/template.rb"
 
+# Javascript code linting and formatting
+run "yarn add --dev eslint eslint-plugin-prettier eslint-config-prettier eslint-plugin-eslint-comments eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-jsx-a11y prettier"
+copy_file ".eslintrc.js"
+copy_file ".prettierrc"
+run "./node_modules/.bin/eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts,tsx,jsx --fix"
+
+package_json = JSON.parse(File.read("./package.json"))
+package_json["scripts"] = {
+  "lint" => "eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts,tsx,jsx",
+  "lint-fix" => "eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts,tsx,jsx --fix"
```

Comment:
🐘 Ideally we'll also want to add `"format": "prettier --write \"./**/*.{css,scss,json,md}\"",` at some point (those won't be handled by `eslint --fix`).

Reply:
will add